tests/product/base_product_case.py

```diff
@@ -131,12 +131,18 @@ def _run_installers(self, installers):
             installer_instance = installer(self)
             installer_instance.install()
 
+            self.default_keywords.update(installer_instance.get_keywords())
             cluster.postinstall(installer)
 
     def _apply_post_install_hooks(self, installers):
         for installer in installers:
             self.cluster.postinstall(installer)
 
+    def _update_keywords(self, installers):
```

Comment:
it looks like this name existed before, but could you call this something more descriptive than keywords.  It's not clear what the keywords are for.